src/commands/account/get_public_key/from_seed_phrase/mod.rs

```diff
@@ -0,0 +1,28 @@
+use std::str::FromStr;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = PublicKeyFromSeedPhraseContext)]
+pub struct PublicKeyFromSeedPhrase {
+    /// Enter the seed-phrase:
+    master_seed_phrase: String,
+}
+
+#[derive(Debug, Clone)]
+pub struct PublicKeyFromSeedPhraseContext;
+
+impl PublicKeyFromSeedPhraseContext {
+    pub fn from_previous_context(
+        _previous_context: crate::GlobalContext,
+        scope: &<PublicKeyFromSeedPhrase as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        let seed_phrase_hd_path_default = slipped10::BIP32Path::from_str("m/44'/397'/0'").unwrap();
```

Comment:
I believe, we could configure BIP32Path in other handlers

Reply:
I think we should stay consistent here

Reply:
[Example](https://github.com/near/near-cli-rs//blob/main/src/commands/account/import_account/using_seed_phrase/mod.rs#L11). It actuallyt seems to be skipped, but we can at least have a structure similar way